srcpkgs/mesa/template

```diff
@@ -23,7 +23,7 @@ license="MIT, LGPL-2.1-or-later"
 homepage="https://www.mesa3d.org/"
```

Comment:
Would you mind checking if replace `libxml2-python` with `libxml2-python3` in `hostmakedepends` and replace Juan's name with something else, Orphan is a good candidate?

Reply:
it seems `mesa` doesn't even depend on it anymore, so I can just remove it. I think @q66 is the de-facto maintainer, I can put his name if he wants

Reply:
I've tested builds against libxml2-python3 last 2-3 releases, 
have not noticed any breakage, well atleast locally on x86_64

Reply:
feel free to add me

Reply:
it seems that it was originally employed in mapi/glapi and later removed:
https://gitlab.freedesktop.org/mesa/mesa/-/commit/2de70fe23f320ce4f559e37057fe07b7af99cf5a
I guess it's safe to remove then

Reply:
we should add `libxml2-devel` to `makedepends` though, since `freedreno` seems to need it to build some components

Reply:
Yeah, I've seen it. And it checks for `lua-devel`, `libarchive-devel` and `gzip` too. We could add them all to the `_have_arm` section

Reply:
nah, i'd just add it normally, there's no point in conditionalizing it